client/src/main/java/com/alibaba/nacos/client/config/impl/ServerListManager.java

```diff
@@ -154,8 +155,9 @@ public ServerListManager(Properties properties) throws NacosException {
         if (StringUtils.isNotEmpty(serverAddrsStr)) {
```

Comment:
Hi @mask616 , if we can add the unit test to test this new change like com.alibaba.nacos.client.config.impl.ServerListManagerTest#testGetter, I think it will be better.

Reply:
Done